src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_References/Arrays_Ex1.java

```diff
@@ -0,0 +1,35 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_References;
+
+public class Arrays_Ex1 {
```

Comment:
Could you please find a more appropriate class name to be more representable for the code written in it

Reply:
Done